data/remote/airspace/country/CN-ASP-National-Soaringweb.txt.json

```diff
@@ -1,5 +1,5 @@
 {
   "description": "Canada Airspace from Soaringweb.org",
-  "update": "2024-12-26",
-  "uri": "https://soaring.silentflight.ca/Airspace/NA/CanAirspace311all.txt"
+  "update": "2025-06-11",
```

Comment:
- The current file at https://soaring.silentflight.ca/Airspace/NA/CanAirspace314all.txt is effective 2025-04-17.
- Is this `update` field meant to be the date when the airspace file is effective, or the date when we added it?
- I always thought it should reflect when the airspace file became effective. If so, we can either wait for the next update which should be 2025-06-12 according to the current file, or we can change this `update` field to `2025-04-17` and merge this.